contracts/KNS_Retirer_v1.sol

```diff
@@ -12,17 +12,15 @@ import "./interfaces/ITreasury.sol";
 import "./interfaces/IUniswapV2Pair.sol";
 import "./interfaces/IUniswapV2Router02.sol";
 
-
 // KNS Retirer retires 5% of mint price as BCT, wraps 5% into
 // auto-retirement KLIMA that retires 33% of any staking rewards
 // and is permanently locked in the contract, and transfers the
 // remainder to GWAMI Labs treasury. 
 contract KNS_Retirer is Initializable, OwnableUpgradeable {
 
     // EVENTS
-    event maticReceived(address indexed _sender, uint value );
-    event BCT_retired( address indexed _beneficiary, uint BCTFromSwap );
-    event arKLIMAMinted( uint _value );
```

Comment:
This event was removed because I'm pretty sure wrapping is tracked in the arKLIMA wrapping contract